src/integrationTest/java/com/mongodb/hibernate/JakartaPersistenceBootstrappingIntegrationTests.java

```diff
@@ -44,8 +45,8 @@ static void beforeAll() {
         mongoClient = MongoClients.create(config.mongoClientSettings());
     }
 
-    @BeforeAll
-    static void beforeEach() {
+    @BeforeEach
+    void beforeEach() {
```

Comment:
This was a mistake, the method was supposed to be annotated with `@BeforeEach`, not `@BeforeAll`.